frontend/packages/kubevirt-plugin/src/selectors/vm/flavor-text.ts

```diff
@@ -19,5 +20,5 @@ export const getFlavorText = ({
   const memoryBase = convertToBytes(memory);
   const memoryText = humanizeBinaryBytes(memoryBase).string;
 
-  return `${_.capitalize(flavor) || ''}${flavor ? ': ' : ''}${vcpusText}, ${memoryText} Memory`;
+  return `${_.capitalize(alignLegacyFlavor(flavor))}: ${vcpusText}, ${memoryText} Memory`;
```

Comment:
do we need to check for "Custom" here ?

If it's the legacy "Custom", turning it to "custom" before turning it back to "Custom" does nothing.

Reply:
This is mainly to show Custom even when the flavor is missing. 

This begs a question: shouldn't we remove the custom label altogether and treat missing flavor label as custom every time?

I mean this is already happening, but we probably don't need two approaches to this.

what do you think?

Reply:
+1 i'm for removing the custom label ...
are we the only one's using this label ?

Reply:
cool, will change the PR then,

yes, we are although vm-import-operator started using it too, but it should be easy for them to drop it